core/services/userServices.py

```diff
@@ -233,6 +233,14 @@ def create_or_update_core_user(user_uuid, username, i_user=None, t_user=None, of
 
 
 def change_user_password(logged_user, username_to_update=None, old_password=None, new_password=None):
+    print("-------------------------------")
+    print("-------------------------------")
+    print("-------------------------------")
+    print(old_password)
+    print(new_password)
+    print("-------------------------------")
+    print("-------------------------------")
+    print("-------------------------------")
```

Comment:
Remove this

Reply:
yeah thanks, it should be draft PR that is why it is here